src/useChat.ts

```diff
@@ -34,11 +33,11 @@ type UseChatConfig = {
 
 export const useChat = ({
   customerId,
-  corpusIds,
+  corpusKey,
   apiKey,
-  enableStreaming = true,
+  numberOfSearchResults = 15,
```

Comment:
Let's be sure to document this in the README.
Also, is there any reason for the default value of 15 here?

Reply:
No reason. I just picked that 15 results should be enough for a good summary. If we have a preference I can change it.

Reply:
@ofermend Can you suggest a good number here? Will 15 risk blowing the token limit?

Reply:
Historically we used about 5, but that was mostly due to the limits in sequence length for GPT-3.5-Turbo. Nowadays, I expect most customers to use GPT-4o which has a much larger limit, so I am not so concerned about that. However, 15 still seems too high for default, maybe use 10?